padglobal/padglobal.py

```diff
@@ -1131,12 +1131,12 @@ async def whichinvade(self, ctx):
         """Display which yinyangdra is currently invading for Mystics & Spectres event"""
         pst = pytz.timezone("America/Los_Angeles")
```

Comment:
You know, you could probably do this with a timezone that doesn't honor DST and have it work year round.  America/Phoenix uses this, but there's probably some way to say that you want PST and not PT.

Reply:
`Etc/GMT+8` is a defined timezone for GMT-8 (don't ask me why it's called GMT+8.   (If you really want to know you can look here https://en.wikipedia.org/wiki/Tz_database#Area))

Reply:
I'm merging this for now so it's not broken in prod but we can open another PR further improving the logic